net-ftp/pure-ftpd/files/pure-uploadscript.initd

```diff
@@ -0,0 +1,30 @@
+#!/sbin/openrc-run
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+ftpd_configfile="/etc/${SVCNAME/uploadscript/ftpd}.conf"
+pidfile="/run/pure-ftpd.pid"
```

Comment:
Should this init script really use the same pidfile like the `pure-ftpd.initd-r12` does?

Reply:
Indeed, a copy/paste error. Fixed now. Thanks!